packages/apps/shopify-app-remix/src/server/authenticate/admin/helpers/__tests__/redirect.test.ts

```diff
@@ -173,59 +213,52 @@ describe('Redirect helper', () => {
       );
     });
 
-    it('parses shopify admin routes', async () => {
+    it('returns an app bridge script on bounce requests', async () => {
```

Comment:
I just moved a couple of tests around because they were in the "wrong" `describe` block, but they're untouched.